lib/flash_patch/CMakeLists.txt

```diff
@@ -4,7 +4,8 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-if((NOT CONFIG_DISABLE_FLASH_PATCH) AND CONFIG_SOC_NRF52840
+if((NOT CONFIG_DISABLE_FLASH_PATCH)
+    AND (CONFIG_SOC_NRF52840 OR CONFIG_SOC_NRF52833)
```

Comment:
We should pull this out to a helper kconfig (promptless) so that we don't need this logic in CMake, but instead could check `if ( CONFIG_FLASH_PATCH_WARN) ` where

```
config FLASH_PATCH_WARN
   bool
   depends on SOC_NRF52840 || SOC_NRF52833
   depends on !DISABLE_FLASH_PATCH
   depends on IS_SECURE_BOOTLOADER || MCUBOOT
   default y
```

Reply:
Good idea, done.